src/pybind/mgr/devicehealth/module.py

```diff
@@ -14,10 +15,19 @@
 TIME_FORMAT = '%Y%m%d-%H%M%S'
 
 DEFAULTS = {
-    'enable_monitoring': True,
+    'enable_monitoring': str(True),
     'scrape_frequency': str(86400),
     'retention_period': str(86400*14),
     'pool_name': 'device_health_metrics',
+    'mark_out_threshold': str(86400*14),
+    'warn_threshold': str(86400*14*2),
+    'self_heal': str(True),
+}
+
+health_messages = {
```

Comment:
Constants should be like `HEALTH_MESSAGES`.

To cut down on typos, I also like to make enum-ish strings into constants, to avoid risk of typos later (i.e. do DEVICE_HEALTH = 'DEVICE_HEALTH' etc).  It's a little wonky, but python 2 doesn't give us real enums